### PR TITLE
feat(dance): add font-size dance

### LINKED ISSUE #66: Font-size Dance
Make the `font-size` dance !
Options : min & max

### PR DESCRIPTION
**Pull request by Slashgear**
Fix #66

Here is my contribution for adding `font-size` dance.
I played with `em` size to be relative of the default size.

Should I had an entry in the documentation ?

### CODE REVIEW

**Comment by Okazari** [comment]
Thanks for the PR !
LGTM
Yes you should add documentation for your dance (options and defaults) :)

Could you also provide a small gif showing off you dance ? 
I'll use it to promote your new dance on Twitter :)

**Comment by Slashgear** [comment]
font-size

File: README.md [hunk 1]
```diff
@@ -193,6 +193,9 @@ Here are the build in dances and their options
 + borderWidth
   + min : Minimum value given to `border-width`. Default: `0`
   + max : Maximum value given to `borderr-width`. Default: `5`
++ fontWidth
```
Comment (Okazari) [inline README.md]: You named it `fontSize` in the code ;)

Comment (Slashgear) [reply README.md]: Ok, gonna fix it 🤣 

Did you see the borderr just above ? Do you want me to also fix it ?

Comment (Okazari) [reply README.md]: Oh ! Didn't see this one, if you want to fix it, i would prefer to have a separated PR for that fix if you don't mind ;)

Comment (Slashgear) [reply README.md]: Ok I'm gonna PR Farm 🤠

**Pull request merged**